services/basic_types.proto

```diff
@@ -1623,21 +1628,9 @@ message GrantedNftAllowance {
     TokenID token_id = 1;
 
     /**
-     * The account ID of the token allowance spender.
+     * The account ID of the spender that has been granted access to all NFTs of the owner
      */
     AccountID spender = 2;
-
-    /**
-     * The list of serial numbers that the spender is permitted to transfer.
-     */
-    repeated int64 serial_numbers = 3;
-
```

Comment:
Shouldn't we also return the new delegating_spender field?

Reply:
All spenders that have approve_for_all allowance grant can be delegating_spenders. It will be redundant to have to this field here.